src/components/users/UserImageName.js

```diff
@@ -1,20 +1,29 @@
 import React from 'react';
-import ExampleComponent from 'react-rounded-image';
 
 import defaultProfileImg from '../../assets/profilePicture.png';
 import './user-profile.scss';
 import { userShape } from '../../constants/shapes';
 
 const UserImageName = ({ user }) => {
+  let showImage = defaultProfileImg;
+  if (user.profile_picture) {
+    showImage = user.profile_picture;
+  }
+
   return (
     <div className="user-profile__profile-picture">
-      <div>
-        <ExampleComponent
-          image={defaultProfileImg}
-          roundedSize="0"
-          imageWidth="150"
-          imageHeight="150"
-        />
+      <div
+        style={{
+          width: 150,
+          height: 150,
+          borderRadius: 100,
+          overflow: 'hidden',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <img src={showImage} width="250" alt="user" />
```

Comment:
```suggestion
        <img src={user.profile_picture || defaultProfileImg} width="250" alt="user" />
```

I also think there is no need to set the width. Have you tried?